package.json

```diff
@@ -40,6 +40,7 @@
     "pino": "^8.15.3",
     "pino-pretty": "^10.2.0",
     "soroban-client": "^1.0.0-beta.2",
+    "stellar-sdk": "^10.4.1",
```

Comment:
i know it's still in beta - but should we use the latest release? https://github.com/stellar/js-stellar-sdk/releases/tag/v11.0.0-beta.6

Reply:
oh yeah probably so and I actually need to drop soroban client as well so I will do both of those.

Reply:
done in 221c740faf710c87a355ed555dfc4c523c630058